ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -284,8 +296,10 @@ def start_learning(
         env_manager.close()
         if self.train_model:
             self._write_training_metrics()
+            self._write_timing_tree()
```

Comment:
We might want to write the timing tree even when we're not training (i.e., doing inference). I think if we just move it out of this if statement it will do that..

Reply:
Done